grammar/abnf-grammar.txt

```diff
@@ -1027,6 +1027,13 @@ function-inputs = function-input *( "," function-input )
 
 function-input = [ %s"const" ] identifier ":" type
 
+; A circuit member constant declaration
```

Comment:
The 'consists of' got dropped, it should be:

A circuit member constant declaration consists of